package.json

```diff
@@ -13,7 +13,9 @@
     "demo",
     "packages/*"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "spdx-license-list": "^6.4.0"
+  },
```

Comment:
I think this dependency needs to be in the `packages/elements-core/package.json` in order for the final package to work properly. The top-level `package.json` should only contain `devDependencies` needed for people who clone the elements repo.

Reply:
Hopefully addressed with abc8aac